installer/create-cluster-eks.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+# Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage:
+# create-cluster-eks.sh [EKS-CLUSTER-NAME] [AWS_DEFAULT_REGION]
+if [ $# -ne 2 ]; then
+  echo "Not enough arguments supplied"
+  echo "Usage: create-cluster-eks.sh [EKS-CLUSTER-NAME] [AWS_DEFAULT_REGION]"
+  exit 1
+fi
+
+export EKS_CLUSTER_NAME=$1
+export AWS_DEFAULT_REGION=$2
+
+# Create cluster
+# https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html
+ZONE_1A=a
+ZONE_1B=b
+ZONE_1D=d
```

Comment:
When launching with ```$ ./create-cluster-eks.sh stavros-test2 eu-west-1
[ℹ]  eksctl version 0.12.0
[ℹ]  using region eu-west-1
[ℹ]  subnets for eu-west-1a - public:192.168.0.0/19 private:192.168.96.0/19
...
``` I got:
```[✖]  AWS::EC2::Subnet/SubnetPublicEUWEST1D: CREATE_FAILED – "Value (eu-west-1d) for parameter availabilityZone is invalid. Subnets can currently only be created in the following availability zones: eu-west-1a, eu-west-1c, eu-west-1b. (Service: AmazonEC2; Status Code: 400; Error Code: InvalidParameterValue; Request ID: 4d9aea30-cd81-4101-8f1f-eb857399e7a8)"
````
We need a better approach to pick up the right zones.

Reply:
Fixed